src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlServerFhirDataStore.cs

```diff
@@ -176,7 +180,24 @@ private void PopulateUpsertResourceCommand(SqlCommandWrapper sqlCommandWrapper,
                     requestMethod: resource.Request.Method,
                     searchParamHash: resource.SearchParameterHash,
                     rawResource: stream,
-                    tableValuedParameters: _upsertResourceTvpGeneratorVLatest.Generate(new List<ResourceWrapper> { resource }));
+                    tableValuedParameters: _upsertResourceTvpGeneratorVLatest.Generate(new List<ResourceWrapper> { resource }),
+                    isResourceChangeCaptureEnabled: isResourceChangeCaptureEnabled);
+            }
+            else if (_schemaInformation.Current >= SchemaVersionConstants.SearchParameterHashSchemaVersion)
```

Comment:
nit: Should it just be equality operator (schemaInformation.Current == SchemaVersionConstants.SearchParameterHashSchemaVersion)
since it seems this block will only be executed at schema version: 13.

Reply:
No, it doesn't. the SchemaVersionConstants.SearchParameterHashSchemaVersion indicates SchemaVersion.V8.